app/compliance/management/commands/pull_compliance_list.py

```diff
@@ -0,0 +1,72 @@
+'''
+    Copyright (C) 2019 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+'''
+
+import urllib.request
+import xml.etree as etree
+import xml.etree.ElementTree as ET
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from compliance.models import Country, Entity
+
+
+def insert_countries():
+    # clear existing table
+    Country.objects.all().delete()
+
+    # pull data
+    countries = 'Balkans, Belarus, Burma, Cote D\'Ivoire , Cuba, Democratic Republic of Congo, Iran, Iraq, Liberia, North Korea, Sudan, Syria, Zimbabwe'.split(',')
+
+    # insert data
+    for country in countries:
+        Country.objects.create(name=country)
+
+
+def insert_entities():
+    # clear existing table
+    Entity.objects.all().delete()
```

Comment:
If we're going to ever do any kind of tracking / recording of compliance related service denial or anything like that we'll have to not rewrite this table over and over - a premature optimization for now but lets all make sure we are aware of this for the future.

Reply:
got it; @alexvotofuture should we keep a record of OFAC denials?  would it be handy to have to show regulatory compliance?

Reply:
lets punt to a v2